pom.xml

```diff
@@ -6,15 +6,15 @@
 		<version>2.3.4.RELEASE</version>
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
-	<groupId>me.itzg</groupId>
-	<artifactId>tsdb-cassandra</artifactId>
+	<groupId>com.rackspace.monplat</groupId>
```

Comment:
I kinda expected this to be `com.rackspace.ceres` as i figured over time there would be more services that fall under that group... but I have no strong opinions.

Reply:
I didn't feel strongly either. I'll change that.